nanomq/apps/broker.c

```diff
@@ -1165,7 +1165,9 @@ broker(conf *nanomq_conf)
 		}
 	} while (all_signals[i++] != SIGTERM);
 #endif
+#endif
 
+#if (defined DEBUG) && (defined ASAN)
```

Comment:
for finding memory leakage?

Reply:
basically the logic for tests is disabled now, it shows up as memleak.

Reply:
ok
understood. need a way to enable asan while developing